src/client/Microsoft.Identity.Client/ApiConfig/AcquireTokenInteractiveParameterBuilder.cs

```diff
@@ -115,6 +115,24 @@ public AcquireTokenInteractiveParameterBuilder WithSystemWebViewOptions(SystemWe
             return this;
         }
 
+        /// <summary>
+        /// Specifies options for using the embedded wevbiew for interactive authentication.
+        /// </summary>
+        /// <param name="options">Data object with options</param>
+        /// <returns>The builder to chain the .With methods</returns>
+        #if !SUPPORTS_WEBVIEW2 // currently only WebView2 allows customization
+        [System.ComponentModel.EditorBrowsable(System.ComponentModel.EditorBrowsableState.Never)]
+        #endif
+        public AcquireTokenInteractiveParameterBuilder WithEmbeddedWebviewOptions(
```

Comment:
@jmprieur @jennyf19 - based on @mjcheetham 's feedback, I decided to rename this to `WithEmbeddedWebviewOptions` to match the existing `WithSystemWebViewOptions`

The actual options only affect WV2 and document this.